src/components/CameraError/style.css

```diff
@@ -59,7 +60,7 @@
   .errorOverlay();
   bottom: 0;
   &::before {
-    background-color: #1C1F21;
+    background-color: @color-camera-error-overlay;
```

Comment:
[Minor] Would use more generic naming.

Reply:
I've used these names to make it easier for the designer to find the classes that use those colors. Charlotte passed me a list of variables names and a mapping with the new colors, will fix in the next PR.

Reply:
Actually, a lot of these variables needs to be deleted based on Charlotte's feedback. I will update here